easybuild/easyconfigs/s/strelka/strelka-2.9.7-intel-2018a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'CMakeMake'
+name = 'strelka'
```

Comment:
@verdurin please add a blank line above, to match style used in other easyconfigs